alert/meta.go

```diff
@@ -0,0 +1,26 @@
+package alert
+
+const TypeAlertMetaV1 = "alert_meta_v1"
+
+type AlertMeta struct {
+	Type        string        `json:"type"`
+	AlertMetaV1 AlertMetaData `json:"alert_meta_v1"`
+}
+
+type AlertMetaData map[string]string
```

Comment:
Metadata is one word, also because we're in the context of the `alert` package `Alert` stutters, so this should just be `Metadata`

```suggestion
type Metadata map[string]string
```